#testar se parametros que recebem listas e strings estao certos

CarFreedom_Platform: 'local'
CarFreedom_Component: 'carfreedom'
CarFreedom_Provides_CarFreedom_Name: 'carfreedom'
CarFreedom_Provides_Odometry2d_Name: 'odometry2d'
CarFreedom_Provides_VelocityControl2d_Name: 'velocitycontrol2d'
CarFreedom_Config_CarFreedom_AxisLength_m: 1.15
CarFreedom_Config_CarFreedom_CAN_Delay_Between_Queries_us: 1000
CarFreedom_Config_CarFreedom_CarData_CAN_Timeout_ms: 1
CarFreedom_Config_CarFreedom_CarData_InterfaceCAN: 'can0'
CarFreedom_Config_CarFreedom_Encoder_FrontLeft_Invert: 1
CarFreedom_Config_CarFreedom_Encoder_FrontRight_Invert: 0
CarFreedom_Config_CarFreedom_Encoder_Pulses_Period_ms: 20
CarFreedom_Config_CarFreedom_Encoder_RearLeft_Invert: 0
CarFreedom_Config_CarFreedom_Encoder_RearRight_Invert: 0
CarFreedom_Config_CarFreedom_InterAxesDistance_m: 1.39
CarFreedom_Config_CarFreedom_JoystickMaxForward: 1023
CarFreedom_Config_CarFreedom_JoystickMaxLeft: 0
CarFreedom_Config_CarFreedom_JoystickMaxReverse: 0
CarFreedom_Config_CarFreedom_JoystickMaxRight: 1023
CarFreedom_Config_CarFreedom_PrintCanFrames: 0
CarFreedom_Config_CarFreedom_PulsesPer360: 8192
CarFreedom_Config_CarFreedom_MaxSteerAngleLeft: -20.1607
CarFreedom_Config_CarFreedom_MaxSteerAngleRight: +20.1246
CarFreedom_Config_CarFreedom_SteeringEncoderMaxLeft: 3250
CarFreedom_Config_CarFreedom_SteeringEncoderMaxRight: 3833
CarFreedom_Config_CarFreedom_SteeringAng2CmdStart2ndLine_Adeg_Cmd: [+0.5, 557.36]
CarFreedom_Config_CarFreedom_SteeringAng2Cmd_ZeroCmd: 512
CarFreedom_Config_CarFreedom_WheelRadius_m: 0.335
CarFreedom_Config_CarFreedom_WriteCommandsRPH: 1
CarFreedom_Config_CarFreedomPublishInterval: 0.02
CarFreedom_Config_CarHeartbeat_CAN_Timeout_ms: 1
CarFreedom_Config_DriveInReverse: 0
CarFreedom_Config_DriverLib: 'libHydroCarFreedom.so'
CarFreedom_Config_EnableMotion: 1
CarFreedom_Config_Fake_SleepInterval: 100
CarFreedom_Config_MinPeriod_ms: 25.0
CarFreedom_Config_Odometry2dPublishInterval: 0.1
CarFreedom_Config_TimeOutToZeroCommand_s: 0.3
CarFreedom_Config_VehicleDescription_Control_Type: 'VelocityBicycle'
CarFreedom_Config_VehicleDescription_Control_VelocityBicycle_MaxForwardSpeed: 10.0
CarFreedom_Config_VehicleDescription_Control_VelocityBicycle_MaxReverseAcceleration: 1.0
CarFreedom_Config_VehicleDescription_Control_VelocityBicycle_MaxReverseSpeed: 10.0
CarFreedom_Config_VehicleDescription_Control_VelocityBicycle_MaxSteerAngleAtMaxSpeed: 15.0
CarFreedom_Config_VehicleDescription_Geometry_Cylindrical_Height: 1.0
CarFreedom_Config_VehicleDescription_Geometry_Cylindrical_PlatformToGeometryTransform: [0.0, 0.0, 0.35, 0.0, 0.0, 0.0]
CarFreedom_Config_VehicleDescription_Geometry_Cylindrical_Radius: 0.25
CarFreedom_Config_VehicleDescription_Geometry_Type: 'Cylindrical'
CarFreedom_Config_Odom_Roda_Node_Cycle: 100
CarFreedom_watchdog_duration: 2.0
